src/main/java/org/folio/linked/data/mapper/dto/monograph/instance/InstanceMapperUnit.java

```diff
@@ -109,6 +110,7 @@ public Resource toEntity(Object dto, Resource parentEntity) {
     coreMapper.addOutgoingEdges(instance, InstanceRequest.class, instanceDto.getCarrier(), CARRIER);
     coreMapper.addOutgoingEdges(instance, InstanceRequest.class, instanceDto.getCopyright(), COPYRIGHT);
     coreMapper.addOutgoingEdges(instance, InstanceRequest.class, instanceDto.getWorkReference(), INSTANTIATES);
+    coreMapper.addOutgoingEdges(instance, InstanceRequest.class, instanceDto.getIllustrations(), ILLUSTRATIONS);
```

Comment:
Illustrations is an outgoing edge of Work. 
So, this change should be made in WorkMapperUnit.

Similarly, Changes made in "InstanceRequest.json" and "InstanceResponse.json" has to be moved to "WorkRequest.json" and "WorkResponse.json"